### ISSUE TITLE
Transform tool: input for multiple pairs of coordinates

### ISSUE THREAD

**Issue by klokan** [klokantech/epsg.io#115]
Probably a long textarea replacing the two input boxes - with parsing of text done in JavaScript via regular expression.

Inspiration in: http://cs2cs.mygeodata.eu/

The "Switch X-Y" and "Include input coordinates" is a good idea as well.

**Comment by petrsloup** [comment]
First version of regular expression to be used for the coodinate extraction:
https://regex101.com/r/jR1uN1/2

**Comment by klokan** [comment]
Geodetic and GPS are not yet picked, right?

**Comment by petrsloup** [comment]
No, not yet. Might be a bit difficult to do all this with one regular expression.
